src/Sockets/QMPClient.cpp

```diff
@@ -246,6 +246,13 @@ void QMPClient::OnReadLine(const boost::system::error_code& ec, size_t size, std
 				buf_.consume(size);
 				DoReadLine(ctx);
 				break;
+				}
+				case ConnectionState::kConnecting: {
+					break; // What should this do?
+				}
+				case ConnectionState::kDisconnected: {
+					break; // implode()? murder() death() kill()? i feel like this should restart the VM
+				}
```

Comment:
fwiw the break would have allowed other cases without fall-through

Reply:
Did it? I kept those in anyway in case we wanted to add stuff for either, which is why i left them break-empty

Reply:
well no I mean you didn't need to brace the existing one, it wouldnt have fell through or whatever

Reply:
i'm so use to cases being strictly in blocks that I forgot you can do
```c++
case example:
stuff("example function");
break;
```
and have it work